packages/sanity/src/structure/comments/src/components/list/CommentsListItemContextMenu.tsx

```diff
@@ -9,9 +9,22 @@ import {
   MenuItem,
   TOOLTIP_DELAY_PROPS,
 } from '../../../../../ui-components'
-import {CommentStatus} from '../../types'
+import {CommentReactionOption, CommentReactionShortNames, CommentStatus} from '../../types'
+import {CommentReactionsMenuButton} from '../reactions'
+import {COMMENT_REACTION_OPTIONS} from '../../constants'
+import {ReactionIcon} from '../icons'
 import {ContextMenuButton} from 'sanity'
 
+const renderMenuButton = ({open}: {open: boolean}) => (
+  <Button
+    aria-label="Add reaction"
+    icon={ReactionIcon}
+    mode="bleed"
+    selected={open}
+    tooltipProps={{content: 'Add reaction'}}
```

Comment:
I think this should be added as a translation

Reply:
Nothing is translated in the comments feature – we will treat that as a separate task and translate all text!